capella_model_explorer/__main__.py

```diff
@@ -103,8 +93,9 @@ def run_local():
                 "*.j2",
             ]
         )
+
```

Comment:
My aim is to avoid unnecessary empty lines. I am not there yet and at the same time I wish that we avoid them.

Reply:
I added the empty line to better the readability.